assets/scss/components/_product.scss

```
@@ -0,0 +1,98 @@
+.product {
+  @include flex($justify: space-between, $direction: column, $align: center, $gap: 1rem);
+  padding: 1rem 2rem;
+  position: relative;
+
+  &:hover {
+    .product-hide {
+      opacity: 1;
+    }
+  }
+  &-state {
+    &::after {
+      content: "";
+      width: 0;
+      height: 0;
+      border-top: 0px solid transparent;
+      border-bottom: 50px solid transparent;
+      position: absolute;
+      top: 0;
+      left: 0;
+    }
+    &::before {
+      content: "";
+      width: calc(50px * 1.41);
+      height: 1px;
+      background-color: black;
+      position: absolute;
+      top: calc(#{$square-side} / 2 + 0.25rem);
+      left: calc(#{$square-side} / 2 + 0.25rem);
+      transform: translate(-50%, -50%) rotate(-45deg);
+    }
+    &--new {
+      &::after {
+        border-left: $square-side solid $color-primary-light;
+      }
+      &::before {
```

```suggestion
    }
    
    &--new {
      &::after {
        border-left: $square-side solid $color-primary-light;
      }
      
      &::before {
```
Check lại ở file này, mỗi block cách nhau 1 dòng nhé